### DISCUSSION TITLE
gatsyb-source-strapi Collection inside Component in Dynamic Zone

### DISCUSSION THREAD

**Discussion by arsounet** [gatsby-uc/plugins#402] [Q&A]
Hello,

I'm new to gatsby and strapi but in Strapi I have a collection containing a dynamic zone which call a Component containing a Collection. Everything work fine in gatsby except the component collection that is not retrieve. Can anybody help for that?

Using strapi 4.7.1 and gatsby 5.7.0.

**Comment by hrenaud** [comment]
Did you populate it ?

```javascript
{
// ...
dynamiczone :{
  populate: "*",
    yourComponent : {
    // ...
    populate: "*",
    collection:{
      // ...
      populate: "*",
      // ...
    }
  }
}
...
}
```
